EIPS/eip-2327.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 2327
+title: BEGINDATA opcode
+author: Martin Lundfall martin.lundfall@protonmail.com
```

Comment:
Can you fix the email to be encapsulated in `<>`? You can also use github username instead as `(@MrChico)`.